algorithms/gr-dij/javascript/dijkstra_algorithm.js

```diff
@@ -0,0 +1,52 @@
+/*
+* Dijkstra's single source shortest path algorithm in JavaScript.
+ */
+ const dijkstraAlgorithm = function (graph, start) {
```

Comment:
Codacy found an issue: [Function has a complexity of 10. Maximum allowed is 6.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6358206)